capstone-backend/src/main/java/com/google/sps/data/BookList.java

```diff
@@ -0,0 +1,73 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+/**
+ * The Book List object stores userID and bookshelfID.
+ * 
+ * These IDs variables are meant to be fetch from the Frontend, to extract a
+ * User's data using the Google Books API.
+ * 
+ * Additionally, the Book List object also stores collaboratorsIDs.
+ * 
+ * The 'collaboratorsIDs' variable purpose is to allow the implemation of a
+ * sharing feature between Book Lists.
+ * 
+ */
+public final class BookList {
+
+  private final long id = new Random().nextLong();
+  private String bookshelfID = "";
```

Comment:
up here I typically don't assign my variables (eg. private String bookshelfID; and private ArrayList<Long> collaboratorIDs;) and then assign them only in the constructor.

Reply:
Done.